docs/experimental/examples/oauth2/globus_app.rst

```diff
@@ -138,15 +138,20 @@ invocation of your requested method will proceed as expected.
 Manually Running Login Flows
 ----------------------------
 
-While your app will automatically initiate and oversee auth flows as detected, sometimes
-an author may want to explicitly control when an authorization occurs. To manually
-trigger a login flow, call ``GlobusApp.run_login_flow(...)``. This will initiate an auth
-flow requesting new tokens based on the app's currently defined scope requirements, and
-caching the resulting tokens for future use.
+While your app will automatically initiate and oversee login flows as detected,
+sometimes an author may want to explicitly control when an authorization occurs. To
+manually trigger a login flow, call ``GlobusApp.login(...)``. The app will evaluate the
+current scope requirements against available tokens, initiating a login flow if it
+determines that any requirements across any resource servers are unmet. Resulting tokens
+will be cached for future use.
 
-This method accepts a single optional parameter, ``auth_params``, where a caller
-may specify additional session-based auth parameters such as requiring the use of an
-MFA token or rendering with a specific message:
+This method accepts two optional keyword args:
+
+-   ``auth_params`` a collection of additional auth parameters to customize the login.
+    This allows for specifications such as, requiring that a user be logged in with an
+    MFA token or rendering the authorization webpage with a specific message.
```

Comment:
I'd love to link out to another doc here expanding on what can be passed but the two candidates I looked at were:
* [SDK Docs on GAREs](https://globus-sdk-python.readthedocs.io/en/stable/experimental/auth_requirements_errors.html) which don't enumerate the list of valid options.
* [Auth API Docs on Authorize Code Grant](https://docs.globus.org/api/auth/reference/#authorization_code_grant_preferred) which includes additional items beyond those that go strictly into a GAREData object.